debian/control

```diff
@@ -2,10 +2,10 @@ Source: spud
 Maintainer: The ICOM team <patrick.farrell@imperial.ac.uk>
 Uploaders: David Ham <david.ham@imperial.ac.uk>
 Section: science
-Priority: extra
-Build-Depends: gfortran (>=4.2), g++ (>=4.2), python3-setuptools (>= 0.6b3-1), python3-all-dev (>= 2.3.5-11), debhelper (>= 5.0.38), dh-python, texlive
+Priority: optional
+Build-Depends: gfortran (>=4.2), g++ (>=4.2), python3-setuptools (>= 39), python3-pip, python3-all-dev , debhelper (>= 5.0.38), dh-python, texlive
```

Comment:
Same question about pip, is it a required dependency?

Reply:
No indeed. Removed